docs/reference/ilm/apis/put-lifecycle.asciidoc

```diff
@@ -35,7 +35,7 @@ include::{docdir}/rest-api/timeoutparms.asciidoc[]
 
 You must have the `manage_ilm` cluster privilege to use this API. You must
 also have the `manage` index privilege on all indices being managed by `policy`.
-All operations executed by {Ilm} for a policy are executed as the user that
+All operations executed by {ilm} for a policy are executed as the user that
```

Comment:
I assume the lowercase here is intentional.  However, I don't see any uses of `ilm-cap`. (outside the scope of this PR...but should that be removed?)

Reply:
Yes, I didn't see why the "I" was capitalized in this case.